lib/src/store.dart

```diff
@@ -85,8 +85,10 @@ class Store extends Disposable {
       throw new StateError('Store has been disposed');
     }
 
-    return _stream.listen(onData,
```

Comment:
Since this is essentially proxying a `listen` call, I think it would be better to allow the consumer to manage this/have control of their own subscription.